Sources/Contour/IIncomingMessageHeaderStorage.cs

```diff
@@ -18,5 +18,11 @@ public interface IIncomingMessageHeaderStorage
         /// </summary>
         /// <returns>Заголовки входящего сообщения.</returns>
         IDictionary<string, object> Load();
+
+        /// <summary>
+        /// Registers header names to be excluded on storing
+        /// </summary>
+        /// <param name="headers">Header names</param>
+        void RegisterExcludedHeaders(IEnumerable<string> headers);
```

Comment:
The storage is supposed to store and load. If some headers need to be excluded they should be passed via constructor.